### PR TITLE
ICDC-1707

### PR DESCRIPTION
**Pull request by tyudosen**
Fixed error in the headers of .csv files downloaded from the Dashboard,

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "@material-ui/core": "^4.10.0",
     "@material-ui/icons": "^4.9.1",
     "@svgr/webpack": "4.1.0",
-    "axios": "^0.19.1",
+    "axios": "^0.21.1",
```
Comment (rana22) [inline package.json:12]: We can update version later at once. It would be easy to track if issue is related to dependencies.

**Review by rana22 — APPROVED** [review]

**Review by rana22 — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "@material-ui/core": "^4.10.0",
     "@material-ui/icons": "^4.9.1",
     "@svgr/webpack": "4.1.0",
-    "axios": "^0.19.1",
```
Comment (iksheth) [inline package.json:12]: @tyudosen Please revert this change.

**Review by iksheth — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (tyudosen) [reply package.json]: done

**Review by iksheth — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**